components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { LocaleProvider } from '../../../../providers/localeProvider';
```

Comment:
We shouldn't use `LocaleProvider` here, just the plain `IntlProvider`

Reply:
I will test